randovania/games/prime1/patcher/randomprime_patcher.py

```diff
@@ -397,6 +397,7 @@ def create_patch_data(self, description: LayoutDescription, players_config: Play
                 "startingRoom": starting_room,
                 "startingMemo": starting_memo,
                 "warpToStart": configuration.warp_to_start,
+                "springBall": configuration.spring_ball,
```

Comment:
Changes made here need to be reflected in the expected test output used by the randomprime patcher test.